tests/unit/test_migration.py

```diff
@@ -81,7 +86,7 @@ def test_record_files_migration(app, location, script_info, datadir):
     assert record['_files'][0]['doctype'] == 'CTH_FILE'
 
 
-def test_mingrate_pids(app, location, datadir):
+def test_mingrate_pids(app, location, datadir, users):
```

Comment:
typo :P (unless `mingrate` is a clever play with words that means `migrate and mint`) :smile: